_strategies/human-centered-design.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: strategy
+title: Human-Centered Design
+description: Build a product that works well for your users.
+permalink: /human-centered-design/
+tags: strategy
+image: /assets/img/strategy-images/hcd.jpg
+image-credit: https://www.flickr.com/photos/nearsoft/
+---
+
+## What is human-centered design and why is it important in modular contracting?
+
+Human centered design is an approach to engage the end user in software development to yield actionable, prioritized, and assumption-challenging results. By directly engaging users throughout, HCD builds products that work better for users, require less rework, and prompt fewer support calls. We believe that theses approaches need to be incorporated at every step of the agile process. From conducting design research before and at the beginning of an acquistition, and usability testing for every sprint and user story developed.
+
+## Four tenets of human-centered design
+1. Put people before technology.
+2. Conduct design research.
+3. Focus on common user needs.
+4. Only accept proof in use.
+
+### Put people before technology.
+This does not mean that people design the system. In human-centered design we figure out the best way to meet people's needs and then figure out how to make it work within our constraints. If we start with user needs, we can see what needs we can’t meet within our current limitations and work on expanding them. This often maximizes meeting user's needs. Moreover, it helps us prioritize what needs should be met and which needs can be adapted given our constraints.
+
+### Conduct design research.
+We find out people's needs by observing and speaking directly with them with design research methods — not via surrogates or our own assumptions.
```

Comment:
Suggest: "We find out people's needs by observing and speaking directly with them using design research methods..."